### PR TITLE
DX-514 Added onBillingDetailsAvailable under Checkin in Checkout section

### PR DESCRIPTION
**Pull request by trieutran91**

### CODE REVIEW

File: checkout/checkin.md [hunk 1]
```diff
@@ -259,6 +259,16 @@ the following argument objects:
 }
 ```
 
+{:.code-header}
+**Billing Details Available Event Argument Object**
+
+```js
+{
+    "actionType": "OnBillingDetailsAvailable",
+    "url": "https://api.externalintegration.payex.com/psp/consumers/<consumerProfileRef>/billing-details"
```
Comment (asbjornu) [inline checkout/checkin.md]: Do we have examples of what a `GET` request towards `https://api.externalintegration.payex.com/psp/consumers/<consumerProfileRef>/billing-details` returns that we can show?

File: checkout/checkin.md [hunk 2]
```diff
@@ -236,7 +236,7 @@ After that has all loaded, you should see something like this:
 
 As you can see, the payer's information is pre-filled as provided by the
 initial `POST`. When the payer completes the checkin, the events
-`onConsumerIdentified` and `onShippingDetailsAvailable` will be raised with
+`onConsumerIdentified`, `onShippingDetailsAvailable` and `onBillingDetailsAvailable` will be raised with
```
Comment (asbjornu) [inline checkout/checkin.md]:
```suggestion
`onConsumerIdentified`, `onShippingDetailsAvailable` and `onBillingDetailsAvailable` may be raised with
```

File: checkout/checkin.md [hunk 3]
```diff
@@ -236,7 +236,7 @@ After that has all loaded, you should see something like this:
 
 As you can see, the payer's information is pre-filled as provided by the
 initial `POST`. When the payer completes the checkin, the events
```
Comment (asbjornu) [inline checkout/checkin.md]:
```suggestion
initial `POST`. During and on completion of Checkin, the events
```

File: checkout/checkin.md (hunk 1 above)
Comment (trieutran91) [reply checkout/checkin.md]: Got this from Syver (check last commit). Is this something in the area of what you had in mind?

File: checkout/checkin.md [hunk 4]
```diff
@@ -259,6 +259,38 @@ the following argument objects:
...
+**Billing Details Available Event Argument Object**
+
+```js
+{
+    "actionType": "OnBillingDetailsAvailable",
+    "url": "https://api.externalintegration.payex.com/psp/consumers/<consumerProfileRef>/billing-details"
+}
+```
+
+Here is an example of what a GET request might return:
```
Comment (asbjornu) [inline checkout/checkin.md]:
```suggestion
Here is an example of what a `GET` request towards the `url` provided in the event might return:
```

File: checkout/checkin.md (hunk 1 above)
Comment (asbjornu) [reply checkout/checkin.md]: Perfect! Similar examples for all other URLs provided in events would be great, but can be fixed as a part of DX-472 in a separate branch.

Comment (trieutran91) [reply checkout/checkin.md]: Great! Let's wrap this one up :)

**Pull request merged**